tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/modules/ServicePossibleValuesService.java

```diff
@@ -28,15 +30,29 @@
 
 public class ServicePossibleValuesService extends PossibleValuesService
 {
+
     @Override
     public Status problem( final Value<?> value )
     {
         return Status.createOkStatus();
     }
-   
+
     @Override
     protected void compute( final Set<String> values )
     {
-        values.addAll( Arrays.asList( NewLiferayModuleProjectOpMethods.getServices() ) );
+        try
```

Comment:
We have a new solution to resolve this. Lovetii will firstly post his pull request, then I will send my new pull